test/e2e/specs/onboarding/signup__with-theme-LOHP.ts

```diff
@@ -0,0 +1,190 @@
+/**
+ * @group calypso-release
+ */
+
+import {
+	DataHelper,
+	BrowserManager,
+	DomainSearchComponent,
+	UserSignupPage,
+	SignupPickPlanPage,
+	CartCheckoutPage,
+	SecretsManager,
+	NewSiteResponse,
+	RestAPIClient,
+	NewUserResponse,
+	MyProfilePage,
+	MeSidebarComponent,
+	cancelPurchaseFlow,
+	NoticeComponent,
+	PurchasesPage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+import { apiCloseAccount } from '../shared';
+
+declare const browser: Browser;
+
+/**
+ * Checks the entire with theme user lifecycle, from signup, onboarding, launch and plan cancellation.
+ *
+ * Keywords: Onboarding, Store Checkout, Coupon, Signup, Plan, Subscription, Cancel
+ */
+describe( 'Lifecyle: Logged Out Home Page, signup, onboard, launch and cancel subscription', function () {
+	const planName = 'Premium';
+	let themeSlug: string | null = null;
+
+	const testUser = DataHelper.getNewTestUser( {
+		usernamePrefix: 'ftmepremium',
+	} );
+
+	let page: Page;
+	let newUserDetails: NewUserResponse;
+	let newSiteDetails: NewSiteResponse;
+
+	beforeAll( async () => {
+		page = await browser.newPage();
+	} );
+
+	describe( 'Signup', function () {
+		let cartCheckoutPage: CartCheckoutPage;
+		let signupPickPlanPage: SignupPickPlanPage;
+
+		beforeAll( async function () {
+			await BrowserManager.setStoreCookie( page, { currency: 'USD' } );
+		} );
+
+		it( 'Navigate to the Logged Out Home Page', async function () {
+			await page.goto( DataHelper.getCalypsoURL( '/' ) );
+		} );
+
+		it( 'Selects a theme', async function () {
+			const themeContainer = await page.locator( '.lp-content.lp-content-area--scrolling' ).first();
```

Comment:
We can drop the `await` here.

```suggestion
			const themeContainer = page.locator( '.lp-content.lp-content-area--scrolling' ).first();
```

Reply:
Thanks, removed https://github.com/Automattic/wp-calypso/pull/84865/commits/0d8b529dcb3e429c3ab840dda26b72bb515e3992